api/envoy/extensions/filters/http/aws_lambda/v3/aws_lambda.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.http.aws_lambda.v3;
+
+import "udpa/annotations/versioning.proto";
+
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.filters.http.aws_lambda.v3";
+option java_outer_classname = "AwsLambdaProto";
+option java_multiple_files = true;
+
+// [#protodoc-title: AwsLambda]
+// AwsLambda :ref:`configuration overview <config_http_filters_aws_lambda>`.
+// [#extension: envoy.filters.http.aws_lambda]
+
+// AWS Lambda filter config
+message Config {
+  option (udpa.annotations.versioning).previous_message_type =
+      "envoy.config.filter.http.aws_lambda.v2alpha.Config";
+
+  // The ARN of the AWS Lambda to invoke when the filter is engaged
```

Comment:
To avoid confusion about the ARN format, can you include an example somewhere in the docs?

Reply:
Good call.